server/app/services/Address.java

```diff
@@ -0,0 +1,36 @@
+package services;
+
+import com.google.auto.value.AutoValue;
+
+/** Represents a basic address in the context of address correction */
```

Comment:
suggestion (blocking): 
```suggestion
/** Represents a basic address. */
```